index.html

```diff
@@ -37,10 +39,10 @@
                 <a class="menu-desktop" href="#"><img alt="Email"
                         src="assets/mail.png"></a>
             </div>
-            <div class="mobile-header">
-              <h1>Mehria Rahmani</h1>
-                <img class="hamburger" alt="Background Image" src="assets/Union.png">
-        </div>
+            <div class="mobile">
+                <h1 class="logo2">Mehria</h1>
+                <img class="humb" src="./assets/Union.png" alt="">
+              </div>
```

Comment:
- [ ] You added the hamburger button on the desktop version although it should only be present on the mobile version, please fix it to match the design 👇 
![Screenshot from 2022-06-11 13-43-14](https://user-images.githubusercontent.com/87965534/173184538-62ffe094-f0c8-4873-b849-8b6cebea4f1e.png)
Your design 👇 
![Screenshot from 2022-06-11 13-45-06](https://user-images.githubusercontent.com/87965534/173184565-e36864f6-acd9-4fe0-b753-b4d729796f32.png)


Make sure 

- The links are on the right
- Mehria is on the left
- The hamburger button is hidden on the desktop version